packages/cozy-client/src/types.js

```diff
@@ -170,6 +171,11 @@ import { QueryDefinition } from './queries/dsl'
  * @property {object} [schema] - the schema used by prosemirror (with notes and marks serialized as arrays to preserve the order).
  * @property {string} [title] - the initial title of the note (that will also be used for the file name)
  * @property {number} [version] - Number of a note
+ * @property {Qualification} [qualification] - Qualification of the file
+ * @property {string} [country] - Country of the paper
+ * @property {string} [expirationDate] - Expiration date of the paper
```

Comment:
String? Are-you sure?

Reply:
The expiration date is currently set by the user in an date field in Mes Papiers and stored as is in the metadata. So yes this is a string.